modules/rtbhouseBidAdapter.js

```diff
@@ -115,10 +115,11 @@ function mapBanner(slot) {
   if (slot.mediaType === 'banner' ||
     utils.deepAccess(slot, 'mediaTypes.banner') ||
     (!slot.mediaType && !slot.mediaTypes)) {
+    var sizes = slot.sizes || slot.mediaTypes.banner.sizes;
```

Comment:
To wygląda jakby miało krzyczeć jak nie będzie `slot.mediaTypes`

Reply:
W odwrotnej kolejności by krzyczało ;) 
Jest leniwe wyszukiwanie, więc jak znajdzie sizes a nie znajdzie meidaTypes to będzie ok.
No a jak nie będzie ani jednego ani drugiego to chyba nam nie przeszkadza ze się wywali.

Reply:
also sprawdzone w testach bez mediaTypes i żyje :)